Python/ceval.c

```diff
@@ -5056,17 +5057,21 @@ import_from(PyObject *v, PyObject *name)
 
     if (pkgpath == NULL || !PyUnicode_Check(pkgpath)) {
         PyErr_Clear();
-        PyErr_SetImportError(
-            PyUnicode_FromFormat("cannot import name %R from %R (unknown location)",
-                name, pkgname_or_unknown),
-            pkgname, NULL);
+        errmsg = PyUnicode_FromFormat(
+            "cannot import name %R from %R (unknown location)",
+            name, pkgname_or_unknown
+        );
+        /* doesn't check NULL for errmsg, PyErr_SetImportError will catch */
```

Comment:
This comment is to make others know the error check is left out deliberately. It is not an oversight.
IMHO it's useful for now. :-（